src/Elements/Bundle/AlternateObjectTreesBundle/LevelDefinition/Input.php

```diff
@@ -52,15 +52,19 @@ public function getGroupedValues($condition, $offset = null, $limit = null)
         // create condition
         $where = $this->condition == '' ? 1 : $this->condition;
         if ($condition) {
+            $where = 'LIKE \'%'.$where.'%\'';
```

Comment:
Maybe we can rename variables is very hard to understand $where going inside $where and also having $this->condition and $condition

Reply:
the whole thing has to actually be reformatted but i wanna do this in a different ticket

Reply:
is there a ticket for this?

Reply:
not yet, i eventually removed the comment entirely

Reply:
PIM-298

this is where i will do all the formatting changes (and more. the goal is to make the whole structure better)

Reply:
(well... me and whoever wants to join)